index.js

```diff
@@ -65,9 +75,10 @@ function setupCanvas(canvas, vlc, options) {
         return;
     }
 
-    vlc.pixelFormat = vlc.I420;
-    canvas.I420Program = gl.createProgram();
-    var program = canvas.I420Program;
+    vlc.pixelFormat = options.vlcPixelFormat ? options.vlcPixelFormat : vlc.RV32;
```

Comment:
By default, we will use RV32 format. But if for some reason, someone needs the other one (I420) she will be able to specify it through option `vlcPixelFormat`.